packages/vm/src/runBlock.ts

```diff
@@ -438,6 +438,8 @@ async function applyBlock(vm: VM, block: Block, opts: RunBlockOpts): Promise<App
     await accumulateParentBlockHash(vm, block.header.number, block.header.parentHash)
   }
 
+  vm.evm.systemVerkleAccessWitness?.flushCache()
```

Comment:
Out of correctness, could we make that part of a isActivatedEIP(6800) conditional?